flips/README.md

```diff
@@ -0,0 +1,158 @@
+# Flow Improvement Proposals (FLIPs)
+
+The purpose of a FLIP is to engage the Flow community in development by 
+leveraging the collective ideas, insights, and experience of contributors and 
+experts while also communicating design changes broadly.
+
+## Who is involved?
+
+Everyone is welcome to propose and provide feedback on a FLIP.
+
+A **FLIP author** writes a FLIP and is committed to championing it through the process.
+
+A **FLIP sponsor** is any maintainer who sponsors the FLIP and will shepherd it
+through the FLIP review process.
+
+A **review committee** is a group of maintainers who are responsible for the 
+strategic direction of components, subcomponents, and public APIs.
+They have the responsibility to accept or reject the adoption of the FLIP.
+
+## What is a FLIP?
+
+A FLIP is a document that describes a requirement and the proposed changes that
+will solve it. Specifically, the FLIP will:
+
+* be formatted according to the FLIP template
+* be submitted as a pull request to the
+  [onflow/flow/flips](https://github.com/onflow/flow/tree/master/flips) directory
+* be subject to discussion and a review meeting prior to acceptance
+
+## FLIP process
+
+Before submitting a new FLIP, check for prior proposals in the `flips` directory, the [Flow community forum](https://forum.onflow.org/) and ask in the [Discord server](https://discord.gg/flow). The idea may have been proposed before or may be in active discussion. Consider contributing or giving feedback to existing proposals.
+
+If a new proposal is appropriate, propose a rough sketch of the idea in the forum [Flow community forum](https://forum.onflow.org/) and engage in discussion with the community, project contributors, and maintainers, to get early feedback. Continue to expand the rough sketch into a draft using the FLIP template and further refine the proposal on the forums.
+
+After writing the FLIP draft, gather feedback from project contributors and
+maintainers before submitting it. 
+
+Once the FLIP is ready for review:
+
+1. _(Optional)_ Recruit a sponsor from the maintainers of the project for which your FLIP concerns.
+
+   You are not required to recruit a sponsor; this step is entirely optional. 
+   However, a sponsor acts as a valuable resource who can streamline the 
+   review process and increase the likelihood of your FLIP being accepted.
+
+   If you do recruit a sponsor, identify them in the FLIP before posting the PR in step 2.
+
+2. Submit your FLIP as a pull request to this repository ([`onflow/flow`](https://github.com/onflow/flow)).
+
+   Name your FLIP file using the [template](./yyyymmdd-flip-template.md) `YYYYMMDD-descriptive-name.md`, 
```

Comment:
Pedantic: Just wondering about the significance of having the date in the filename? Feels like it kind of dates the content.